app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/Name.cs

```diff
@@ -12,18 +13,16 @@ public interface Name
         string LastName { get; }
 
         [Description("The middle name")]
-        string MiddleName { get; }
+        string SecondName { get; }
```

Comment:
We do not really know the icbc SecondName and ThirdName meaning yet.
And SecondName and ThirdName is not explicit enough.